packages/mui-varnish/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@allenai/mui-varnish",
+  "version": "5.11.0",
+  "private": false,
+  "author": "Reviz",
+  "description": "Varnish theme and components that implement Google's Material Design.",
+  "main": "./src/index.ts",
+  "keywords": [
+    "react",
+    "react-component",
+    "mui",
+    "material-ui",
+    "material design",
+    "varnish"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/allenai/mui-varnish.git",
+    "directory": "packages/mui-varnish"
+  },
+  "license": "MIT",
+  "homepage": "https://mui-varnishj.allenai.org/material-ui/getting-started/overview/",
+  "scripts": {
+    "build": "yarn build:legacy && yarn build:modern && yarn build:node && yarn build:stable && yarn build:umd && yarn build:types && yarn build:copy-files",
+    "build:umd": "cross-env BABEL_ENV=stable rollup -c scripts/rollup.config.js",
+    "prebuild": "rimraf build tsconfig.build.tsbuildinfo",
+    "release": "yarn build && npm publish build",
+    "typescript": "tslint -p tsconfig.json \"{src,test}/**/*.{spec,d}.{ts,tsx}\" && tsc -p tsconfig.json",
+    "typescript:module-augmentation": "node scripts/testModuleAugmentation.js"
```

Comment:
What's module augmentation?

Reply:
in short, it allows you to alter the interface of a lib
im not sure what this specific command does..

but we use it like:

```
interface VarnishMuiTheme {
  color: Dictionary<Color>;
}

interface VarnishMuiThemeOptions {
  color?: Dictionary<Color>;
}

declare module '@mui/material/styles' {
  interface Theme extends VarnishMuiTheme {}

  // allow configuration using `createTheme`
  interface ThemeOptions extends VarnishMuiThemeOptions {}
}
``

Reply:
Interesting. So it modifies the type of something we don't own...